browser/containers/App.tsx

```diff
@@ -114,7 +147,7 @@ export class App extends React.Component<Props, State> {
           </div>
         </nav>
 
-        { generalError != null ? this.mapError(generalError) : '' }
+        { generalError !== null ? this.mapError(generalError) : '' }
```

Comment:
Did that come up on lint? `!=` w/ `null` is actually preferred as it'll also match `undefined`.

Reply:
It did but I am no longer able to get it to repro so I will revert.